Paintroid/src/main/java/org/catrobat/paintroid/command/implementation/RemoveLayerCommand.kt

```diff
@@ -28,8 +28,9 @@ class RemoveLayerCommand(position: Int) : Command {
     var position = position; private set
 
     override fun run(canvas: Canvas, layerModel: LayerContracts.Model) {
-        layerModel.removeLayerAt(position)
-        layerModel.currentLayer = layerModel.layers[0]
+        if (layerModel.removeLayerAt(position)) {
```

Comment:
why this `if` block? 🤔

Reply:
removeLayerAt can fail and will retrung `false` . If it fails, we do not want to update the `currentLayer`

Reply:
hmm.. yeah ok 👍🏻